src/github.com/getlantern/autoupdate/config.go

```diff
@@ -0,0 +1,15 @@
+package autoupdate
+
+const (
+	updateURI = `http://127.0.0.1:9197/update`
+)
+
+type config struct {
+	publicKey []byte
+}
+
+var configMap = map[string]*config{
```

Comment:
Maybe this could be integrated into the application, as we have removed the client and application IDs.